packages/framework/ai-collab/.eslintrc.cjs

```diff
@@ -8,7 +8,10 @@ module.exports = {
 	parserOptions: {
 		project: ["./tsconfig.json", "./src/test/tsconfig.json"],
 	},
-	rules: {},
+	rules: {
+		"unicorn/switch-case-braces": "off",
+		"unicorn/no-array-for-each": "off",
+	},
```

Comment:
I haven't looked at the code yet but I suspect there's no super-compelling reason to disable these instead of following them?

Reply:
This was left over from when we had all the JSON streaming logic, which has since been removed. This config is updated in commit bac309434408ff1fd2adfbc0dcacf19c89f40ee8

Reply:
Looks like these overrides are still here in the most recent commit pushed to the branch.

Reply:
Must have missed this. Should be removed in commit 3177336e7b111b192b05835c8af5d2c3e177f952